packages/server/modules/core/repositories/users.ts

```diff
@@ -80,8 +80,8 @@ export const getUsersFactory =
     userIds = isArray(userIds) ? userIds : [userIds]
 
     const q = tables.users(deps.db).whereIn(Users.col.id, userIds)
-    q.leftJoin(UserEmails.name, UserEmails.col.userId, Users.col.id).where({
-      [UserEmails.col.primary]: true
+    q.leftJoin(UserEmails.name, (j1) => {
+      j1.on(UserEmails.col.userId, Users.col.id).andOnVal(UserEmails.col.primary, true)
```

Comment:
this was causing getUser(s) to return null, even if the user was there, because user_emails is not synced to the region db

Reply:
@gjedlicska test prep. even outside of that, the query is invalid - it should not filter out users just because they dont have a UserEmails entry